source/includes/tutorials/port-mapping-note.rst

```diff
@@ -3,7 +3,7 @@
    The sandbox maps the following services to ports on your host
    machine:
 
-   - The sandbox MongoDB server maps to port ``{+mongodb-port-mapping+}`` on your host machine
+   - The sandbox {+cluster+} maps to port ``{+mongodb-port-mapping+}`` on your host machine
```

Comment:
I was unsure of this instance - does the sandbox use a cluster or a single deployment?

Reply:
I think it is a single node based on the [docker-compose](https://github.com/mongodb-university/kafka-edu/blob/main/docs-examples/mongodb-kafka-base/docker-compose.yml#L98) file.